Lib/sysconfig/__init__.py

```diff
@@ -333,7 +334,12 @@ def get_makefile_filename():
     if _PYTHON_BUILD:
         return os.path.join(_PROJECT_BASE, "Makefile")
 
-    if hasattr(sys, 'abiflags'):
```

Comment:
Are there any details about the existence of `sys.abiflags` for cross-compiling for Windows on Ubuntu? I found it might be non-trivial to replace the following with one another:

```python
hasattr(sys, 'abiflags')
os.name == 'nt'  # os.name != 'posix'
not sys.platform.startswith('win')
```